### PR TITLE
Docker Dev Environment

### PR DESCRIPTION
**Pull request by hskarlupka**
Docker environment for testing pyglidein server/client development.

### CODE REVIEW

File: dev_build/client/root/setup.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+libgomp \
+libstdc++-static \
+perl \
+openssl \
+wget
+
+# Updating ca certs to get epel mirrors to work
+yum -y upgrade ca-certificates --disablerepo=epel
+
+ yum -y groupinstall 'Development Tools'
```
Comment (dsschult) [inline dev_build/client/root/setup.sh]: Nitpick - space at the beginning of the line

File: dev_build/client/root/setup.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+gawk \
+fuse \
+fuse-libs \
+autofs \
+attr \
+gdb \
+policycoreutils-python
+
+rpm -ivh https://ecsft.cern.ch/dist/cvmfs/cvmfs-config/cvmfs-config-default-1.2-2.noarch.rpm
+rpm -ivh --nodeps https://ecsft.cern.ch/dist/cvmfs/cvmfs-2.2.2/cvmfs-2.2.2-1.el6.x86_64.rpm
```
Comment (dsschult) [inline dev_build/client/root/setup.sh]: This seems like a pretty old version of cvmfs.  Any reasoning behind this?

File: dev_build/server/build_server.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#!/bin/bash -e
+rm -f root/pyglidein.tar.gz
+tar -C ../../../ --exclude=pyglidein/dev_build --exclude=pyglidein/.git -czvf root/pyglidein.tar.gz pyglidein
+rm -f root.tar.gz
+cd root; tar czvf ../root.tar.gz .
```
Comment (dsschult) [inline dev_build/server/build_server.sh]: is there a reason to put both commands on one line?

**Comment by hskarlupka** [comment]
@dsschult I added a readme so you can run the tests too.  Let me know if you run into issues.

File: dev_build/client/root/setup.sh (hunk 2 above)
Comment (hskarlupka) [reply dev_build/client/root/setup.sh]: These commands were from a script in the IceCube Wiki.  Is there a suggested version I should move to?

Comment (dsschult) [reply dev_build/client/root/setup.sh]: I think 2.3.5 is the latest version.  I hear 2.4 is in beta.

**Comment by hskarlupka** [comment]
[Setup local pyglidein environment](https://trello.com/c/2I2OnB2n/3-setup-local-pyglidein-environment)

**Comment by hskarlupka** [comment]
[Setup local pyglidein environment](https://trello.com/c/2I2OnB2n/3-setup-local-pyglidein-environment)

**Pull request merged**